resources/functionMap.php

```diff
@@ -8440,58 +8440,62 @@
 'PDO::beginTransaction' => ['bool'],
 'PDO::commit' => ['bool'],
 'PDO::cubrid_schema' => ['array', 'schema_type'=>'int', 'table_name='=>'string', 'col_name='=>'string'],
-'PDO::errorCode' => ['string'],
+'PDO::errorCode' => ['?string'],
 'PDO::errorInfo' => ['array'],
 'PDO::exec' => ['int|false', 'query'=>'string'],
-'PDO::getAttribute' => ['', 'attribute'=>'int'],
+'PDO::getAttribute' => ['mixed', 'attribute'=>'int'],
 'PDO::getAvailableDrivers' => ['array'],
 'PDO::inTransaction' => ['bool'],
-'PDO::lastInsertId' => ['string|false', 'seqname='=>'string'],
+'PDO::lastInsertId' => ['string|false', 'seqname='=>'?string'],
 'PDO::pgsqlCopyFromArray' => ['bool', 'table_name'=>'string', 'rows'=>'array', 'delimiter='=>'string', 'null_as='=>'string', 'fields='=>'string'],
 'PDO::pgsqlCopyFromFile' => ['bool', 'table_name'=>'string', 'filename'=>'string', 'delimiter='=>'string', 'null_as='=>'string', 'fields='=>'string'],
-'PDO::pgsqlCopyToArray' => ['array', 'table_name'=>'string', 'delimiter='=>'string', 'null_as='=>'string', 'fields='=>'string'],
+'PDO::pgsqlCopyToArray' => ['array|string', 'table_name'=>'string', 'delimiter='=>'string', 'null_as='=>'string', 'fields='=>'string'],
 'PDO::pgsqlCopyToFile' => ['bool', 'table_name'=>'string', 'filename'=>'string', 'delimiter='=>'string', 'null_as='=>'string', 'fields='=>'string'],
-'PDO::pgsqlGetNotify' => ['array', 'result_type='=>'int', 'ms_timeout='=>'int'],
+'PDO::pgsqlGetNotify' => ['array|false', 'result_type='=>'int<0,5>', 'ms_timeout='=>'int'],
 'PDO::pgsqlGetPid' => ['int'],
 'PDO::pgsqlLOBCreate' => ['string'],
-'PDO::pgsqlLOBOpen' => ['resource', 'oid'=>'string', 'mode='=>'string'],
+'PDO::pgsqlLOBOpen' => ['resource|false', 'oid'=>'string', 'mode='=>'string'],
 'PDO::pgsqlLOBUnlink' => ['bool', 'oid'=>'string'],
 'PDO::prepare' => ['__benevolent<PDOStatement|false>', 'statement'=>'string', 'options='=>'array'],
-'PDO::query' => ['PDOStatement|false', 'sql'=>'string'],
-'PDO::query\'1' => ['PDOStatement|false', 'sql'=>'string', 'fetch_column'=>'int', 'colno'=>'int'],
-'PDO::query\'2' => ['PDOStatement|false', 'sql'=>'string', 'fetch_class'=>'int', 'classname'=>'string', 'ctorargs'=>'array'],
-'PDO::query\'3' => ['PDOStatement|false', 'sql'=>'string', 'fetch_into'=>'int', 'object'=>'object'],
-'PDO::quote' => ['string', 'string'=>'string', 'paramtype='=>'int'],
+'PDO::query' => ['PDOStatement|false', 'sql'=>'string', 'fetchMode='=>'int<0,5>'],
```

Comment:
You could reference the constants instead: [PDO::FETCH_*](https://www.php.net/manual/en/pdo.constants.php)